packages/core/strapi/lib/types/core/strapi/index.d.ts

```diff
@@ -33,12 +34,12 @@ export interface Strapi {
    *
    * It returns all the registered services
    */
-  readonly services: StringMap<Service>;
+  readonly services: StringMap<GenericService>;
 
   /**
    * Find a service using its unique identifier
    */
-  service<T extends Service = unknown>(uid: string): T | undefined;
+  service(uid: string): GenericService | undefined;
```

Comment:
Would something like this help to allow a specific type when you know what you're looking for?

```suggestion
  service<T extends GenericService = GenericService>(uid: string): T | undefined;
```

Reply:
yep I guess it would help